api/models/Entry.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/config');
+
+
+exports.addEntry = (data) => {
+    const {transaction, category, entry, amount, full_date, created_at, recurring} = data;
+    return db.one('INSERT INTO entries (user_id, transact_id, category_id, entry_desc, amount, full_date, created_at, recurring) VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING *',
+    [2, transaction, category, entry, amount, full_date, created_at, recurring]);
```

Comment:
We can probably get the username from the cookie, fetch the id from the username, and pass the user id from `controllers/auth.js` to `models/Entry.js`

Reply:
if it's stored in state, can't we just pass that?

Reply:
Maybe some bad user can run some JavaScript that changes their username (not sure if this is possible, correct me if I'm wrong). In that case, they can post entries to different users' accounts.

`res.cookie`, on the other hand, can't be messed with using JavaScript.